### ISSUE TITLE
facing issue with setting color to scatterplot

### ISSUE THREAD

**Issue by vijayreddybo** [c3js/c3#1830]
Hello,
I am trying to set a color to a data point on the scatter plot.

i have sample data like: 
<img width="414" alt="data1" src="https://cloud.githubusercontent.com/assets/16921845/18389243/b5f50fe2-76c2-11e6-9da1-67b74592d3f8.png">

I have used Sl_No for x-axis and Quantity for Y-axis. now i want to color the circles based on city.
my data variable looks like the below screenshot.
now when i use 
color : function(color,d){
}
d does not return me any value related to "City". How do i resolve this.

in short, i am expecting to build a chart similar to :+1: http://bl.ocks.org/weiglemc/6185069

<img width="768" alt="data" src="https://cloud.githubusercontent.com/assets/16921845/18389077/ce47e840-76c1-11e6-9591-1cfcc2b5c502.png">

 Any help here is appreciated.
Thanks
VR

**Comment by aendrew** [comment]
Hi there! We try to keep the issue queue clear for specific code-related problems with C3, could you please try asking your question on either the [Google Group](http://groups.google.com/forum/#!forum/c3js) or [Gitter](https://gitter.im/masayuki0812/c3) instead? Thank you!
